main/src/main/java/cgeo/geocaching/settings/fragments/PreferenceMapSourcesFragment.java

```diff
@@ -73,8 +78,11 @@ public void onResume() {
         // display checkbox pref for unified map, if showUnifiedMap is enabled
         final Preference p = findPreference(activity.getString(R.string.pref_fakekey_unifiedmap));
         if (p != null) {
-            p.setVisible(Settings.getBoolean(R.string.pref_showUnifiedMap, false));
+            p.setVisible(Settings.getBoolean(R.string.pref_showUnifiedMap, false) || !BranchDetectionHelper.isProductionBuild());
```

Comment:
Same as above. I would advise usage of `Settings.useUnifiedMap` here

Reply:
Same as above